packages/@aws-cdk/cloud-assembly-schema/CONTRIBUTING.md

```diff
@@ -4,15 +4,33 @@ Making changes to this module should only happen when you introduce new cloud as
 
 > For example: supporting the `--target` option when building docker containers.
 
-If you decided these changes are necessary, simply go ahead and make the necessary modifications to the interfaces that describe the schema.
-Our tests and validation mechanisms will ensure you make those changes correctly.
+If you decided these changes are necessary, simply go ahead and make the necessary modifications to 
+the interfaces that describe the schema. Our tests and validation mechanisms will ensure you make those 
+changes correctly.
+
+### Module Structure
+
+There are two main things to understand about the files in this module:
+
+- [`lib/manifest.ts`](./lib/manifest.ts)
+
+  This is the typescript code that defines our schema. It is solely comprised of structs (property only interfaces).
+  It directly maps to the way we want manifest files to be stored on disk. When you want to make changes to the schema,
+  this is the file you should be editing.
+
+- [`lib/schema.generated`](./schema.generated/cloud-assembly.schema.json)
+
+  This directory contains the generated json [schema](./schema.generated/cloud-assembly.schema.json) from the aformentioned 
```

Comment:
I am not sure I am super fond of the `.generated` suffix. We normally don't commit files with this suffix and this might be confusing to people. I was 100% comfortable with `./schema` and maybe you can add a README.md file in this directory that says "To update, run `yarn update-schema`, DO NOT MODIFY BY HAND". Or something like that.


Having said that, I see that this is now all over the place, so I am okay with leaving it like this :-)

Reply:
I'm also not really fond of this. Added to clarity but since this caught your eye as well im reverting back with a README 👍